automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -351,5 +353,116 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike with
         }
       }
     }
+
+    import DefaultJsonProtocol._
+
+    "should have correct policies in Sam and ACLs in Google when an unconstrained workspace is created" in {
+      implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = 20 seconds)
+      implicit val token: AuthToken = ownerAuthToken
+      val workspacePolicyNames = Set("can-compute", "project-owner", "writer", "reader", "share-writer", "can-catalog", "owner", "share-reader")
+      // bucket and object access levels for sam policies as described in comments in insertBucket function in HttpGoogleServicesDAO
+      val policyToBucketAccessLevel = Map("project-owner" -> "WRITER", "owner" -> "WRITER", "writer" -> "WRITER", "reader" -> "READER")
+      val policyToObjectAccessLevel = Map("project-owner" -> "READER", "owner" -> "READER", "writer" -> "READER", "reader" -> "READER")
+      val serviceAccountEmailDomain = "developer.gserviceaccount.com"
+
+      withCleanBillingProject(owner) { projectName =>
+        withWorkspace(projectName, s"unconstrained-workspace") { workspaceName =>
+          val workspaces = Rawls.workspaces.list()
+          println(workspaces)
```

Comment:
leftover `println` or do you need to log this?

Reply:
leftover! thanks